index.d/fabric8.yml

```diff
@@ -20,3 +20,14 @@ Projects:
     desired-tag: 1.0.1
     notify-email: cp.build.inb@karan.org
     depends-on: centos/centos:latest
+
+  - id: 3
+    app-id: fabric8-hdd-supervisor
```

Comment:
The app-id needs to match the file name. So either have app-id as fabric8, or put the entry in fabric8-hdd-supervisor.yml

Reply:
Ack.
Since the name of container is a combination of app-id, job-id and desired-tag, the HDD stack contains broadly 3 applications :
1. Scouts > 1
2. Supervisor -- The one being worked on now
3. UI
I think it would suffice to have 1 app-id called fabric8-hdd(and accordingly 1 yaml file for all components of HDD stack) with the job-id varying as per the project name